translations/teams.it.xlf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="teams.en.xlf">
```

Comment:
target-language="it"

Reply:
tnk excuse me :-D

Reply:
Change made, next time i will look for missing IT translation in existing files.